artview/components/plot_radar.py

```diff
@@ -83,10 +87,16 @@ def __init__(self, Vradar, Vfield, Vtilt, Vlims=None,
         else:
             self.Vlims = Vlims
 
+        if Vcmap is None:
+            self.Vcmap = Variable(None)
+        else:
+            self.Vcmap = Vcmap
+
         self.sharedVariables = {"Vradar": self.NewRadar,
                                 "Vfield": self.NewField,
                                 "Vtilt": self.NewTilt,
-                                "Vlims": self.NewLims}
+                                "Vlims": self.NewLims,
+                                "Vcmap": self.NewCmap,}
```

Comment:
So this Vcmap is now a shareable variable correct? This way the information could be ported between various windows?